src/main/java/WayofTime/alchemicalWizardry/common/summoning/meteor/MeteorRegistry.java

```diff
@@ -15,10 +16,20 @@ public static void registerMeteorParadigm(MeteorParadigm paradigm) {
         paradigmList.add(paradigm);
     }
 
-    public static void registerMeteorParadigm(ItemStack stack, String[] oreList, int radius, int cost) {
-        if (stack != null && oreList != null) {
-            MeteorParadigm meteor = new MeteorParadigm(stack, radius, cost);
-            meteor.parseStringArray(oreList);
+    public static void registerMeteorParadigm(ItemStack stack, String[] componentList, int radius, int cost) {
+        registerMeteorParadigm(stack, componentList, radius, cost, null, 0);
+    }
+
+    public static void registerMeteorParadigm(ItemStack stack, String[] componentList, int radius, int cost,
+            String[] fillerList, int fillerChance) {
+        if (stack != null && componentList != null) {
+            MeteorParadigm meteor = new MeteorParadigm(stack, radius, cost, fillerChance);
+            meteor.parseStringArray(componentList);
+            if (fillerChance > 0 && fillerList != null && fillerList.length > 0) {
+                meteor.parseStringArray(fillerList, true);
+            } else {
+                meteor.fillerList.add(new MeteorParadigmComponent(new ItemStack(Blocks.stone), 1));
```

Comment:
I am wondering what the right conditions for running this actually should be.

* If `fillerChance` is 0 and `fillerList` is empty (e.g., all current GTNH meteors), you still add a stone block as filler. This will have no observable impact as any place filler is used specifically checks for `fillerChance` being non-zero.
* If `fillerChance` is 0 and `fillerList` is *not* empty, you still overwrite whatever the filler list was in the config with stone. I don't know what the expected behavior should be in this case; but probably not this. (Hypothetically if there was a reagent that increases filler chance by a constant value, rather than multiplicatively, this would lead to the filler defined in config being ignored.)

In my opinion this should be something like:

* `fillerChance > 0`, `fillerList` not empty: Parse `fillerList`.
* `fillerChance > 0`, `fillerList` empty: Add stone.
* `fillerChance <= 0`, `fillerList` not empty: Parse `fillerList`, even though it will not be used.
* `fillerChance <= 0`, `fillerList` empty: Initialize list to some default empty value (null, empty list, stone?)

Reply:
Addressed by [2f96807](https://github.com/GTNewHorizons/BloodMagic/pull/60/commits/2f968072b5053645c33aad79faa00331f991f9fb) by reading the filler list regardless of the value of fillerChance. Setting fillerList to a null/empty list would cause problems with a future reagent that adds a fixed value to fillerChance, so it will just be initialized to stone.

The filler list is not shown in NEI if the filler chance is 0. That information is unused with the current implementation, but it may be useful in the future with a reagent that adds to the filler chance. Should I do something to address that? You can mark this as resolved if not.

Reply:
> The filler list is not shown in NEI if the filler chance is 0. That information is unused with the current implementation, but it may be useful in the future with a reagent that adds to the filler chance. Should I do something to address that? You can mark this as resolved if not.

Looks good; honestly I'm not sure what the "most correct" behavior here would be. If we (or anyone else) decide to give it some meaningful interpretation later, we can always go back and change it.